src/disco/containers/DiscoPane.js

```diff
@@ -96,11 +101,12 @@ export class DiscoPaneBase extends React.Component {
     // TODO: Add captions see https://github.com/mozilla/addons/issues/367
     /* eslint-disable jsx-a11y/media-has-caption */
 
-    const { AddonComponent, results, i18n } = this.props;
+    const { AddonComponent, errorHandler, results, i18n } = this.props;
     const { showVideo } = this.state;
 
     return (
       <div id="app-view" ref={(ref) => { this.container = ref; }}>
+        {errorHandler.hasError() ? errorHandler.renderError() : null}
```

Comment:
I like that. It could even be a good one for William. https://github.com/mozilla/addons-frontend/issues/2929